src/scripts/breeding/Breeding.ts

```diff
@@ -223,7 +223,14 @@ class Breeding implements Feature {
     }
 
     public gainEggSlot() {
-        this.eggSlots += 1;
+        if (this.eggSlots < 4){
```

Comment:
Don't compare with the magic number `4`. Use `this.eggslots.length` instead.

Reply:
I don't actually see how will this change anything??

Reply:
If we leave the 4 there, and decide to allow for more eggslots in the future, it will introduce a bug. This is not the case if you use the eggslots length variable

Reply:
so basically `this.eggSlots < this._eggList.length`?

Reply:
ok, fixed